purchases/src/main/kotlin/com/revenuecat/purchases/common/diagnostics/DiagnosticsAnonymizer.kt

```diff
@@ -6,22 +6,8 @@ internal class DiagnosticsAnonymizer(
     private val anonymizer: Anonymizer,
 ) {
     fun anonymizeEntryIfNeeded(diagnosticsEntry: DiagnosticsEntry): DiagnosticsEntry {
```

Comment:
Note that I currently left the name as `DiagnosticsEntry` instead of renaming to `DiagnosticsEvent`. I think it's ok to keep it as is, but lmk if you prefer a different name.